pkg/services/chgm/chgm.go

```diff
@@ -127,7 +125,7 @@ func InvestigateResolved(r *investigation.Resources) error {
 		})
 		if err != nil {
 			logging.Errorf("Failed to add note '%s' to incident: %s", investigationNotNeededNote, err)
-			return nil
+			return err
```

Comment:
**thought:** I think this was nil because we don't care about the failed note add in this case, but it doesn't hurt to change it to err. I think it makes more sense now :)